### DISCUSSION TITLE
Out of home OTBR Google Home Integration.

### DISCUSSION THREAD

**Discussion by joaoserra** [openthread/openthread#9586] [Q&A]
Hi guys, 

I understand that i can share the Thread Credentials with the Google Home app(https://developers.home.google.com/codelabs/thread-sharing-apis?hl=en#1) so that by Thread devices can be controlled by Google Home via my own OTBR.
I know Matter works only locally but Google nest devices can also act as a hub to allow controlling matter devices out of home via Google Cloud. Is this something that OTBR can do as well?

**Comment by jwhui** [comment]
Yes, any Thread 1.3.0 compliant Thread Border Router can support communicating with Matter-over-Thread devices.

**Reply by joaoserra** [reply]
Hi @jwhui  But i don't understand the cloud part. Google Home can control deices locally but about about outside the home? How can Google Home Cloud initiate connections to the devices? There's NAT and a Firewall in between in home routers.

**Reply by jwhui** [reply]
OTBR is just a Thread Border Router, it does not implement any Matter-specific functionality.
